cypress/components/textarea/textarea.cy.tsx

```diff
@@ -307,6 +307,12 @@ context("Tests for Textarea component", () => {
       textareaChildren().should("be.disabled").and("have.attr", "disabled");
     });
 
+    it("should render Textarea icon with disabled prop", () => {
+      CypressMountWithProviders(<TextareaComponent disabled inputIcon="bin" />);
+
+      textarea().find(ICON).should("be.visible").and("have.attr", "disabled");
```

Comment:
Comment: Not an issue caused by this PR but something I've noticed as a result of this test. We shouldn't be setting `disabled` as an attribute on the Icon span as it's really only meant for form controls. It's being passed as an attribute as styled-components treats lower case props as such. I think it might be better to assert that the icon `should("have.css")` here instead as we should update the Icon component so this isn't being set as an attribute (in a separate ticket).